src/lib.rs

```diff
@@ -85,6 +85,7 @@ macro_rules! directory_entry(
 
 directory_entry!(ExportTable         = RVA<ExportDirectory>);
 directory_entry!(BaseRelocationTable = RVA<RelocationBlock>);
+directory_entry!(Debug               = RVA<DebugDirectory>);
```

Comment:
I think `RVA<[DebugDirectory]>` would be more appropriate. Can you also add `pub fn get_debug_directories(&self) -> Result<&'data [DebugDirectory]>` to `impl Pe`?